src/Settings/Exits/AddExitForm.js

```diff
@@ -23,11 +23,9 @@ const AddExitForm = ({ setAdding, setFillForm }) => {
     try {
       await post("/exits", {
         [identifier]: {
-          id: {
-            mesh_ip: meshIp,
-            eth_address: ethAddress,
-            wg_public_key: wgPubKey,
-          },
+          subnet: meshIp + "/128",
```

Comment:
Since the subnet field is not being used and will be removed anyway ive left it as /128 for now